atm_program.py

```
@@ -0,0 +1,87 @@
+from useraccounts import UserAccounts
+
+CONFIG_FILE_PATH = 'config.json'
+
+
+def get_startup_method(user_accounts, file_path):
+    """
+    Method factory for the startup menu choices.
+    """
+    return {
+        '1': lambda: start_action_loop(user_accounts, file_path),
+        '2': exit_program,
+        '3': user_accounts.create_new_account
+    }
+
+
+def get_startup_choice():
+    """
+    returns the user's startup menu input, after checking that it's a valid choice.
+    """
+    choice = input('\nWelcome! Here are your options:\n\n'
+                   '[1] Log-in\n'
+                   '[2] Exit out of program\n'
+                   '[3] Create a new account\n'
+                   'Enter your choice: ')
+    if choice not in ['1', '2', '3']:
+        print('Invalid choice!')

```

You should just raise valueerror(invalid choice), you do this all around the code, fix it